com.unity.sg2/Editor/GraphDeltaRegistry/FunctionDefinitions/StandardDefinitions/Input/Textures/GatherTexture2DNode.cs

```diff
@@ -0,0 +1,104 @@
+using System.Collections.Generic;
+using Usage = UnityEditor.ShaderGraph.GraphDelta.GraphType.Usage;
+
+namespace UnityEditor.ShaderGraph.Defs
+{
+
+    internal class GatherTexture2DNode : IStandardNode
```

Comment:
When searching for GatherTexture2DNode I get this exception, doesn't break anything else, just wanted to store it somewhere so I can look at it later: 
![image](https://user-images.githubusercontent.com/13033542/167044023-43e239dc-7819-4cad-b4b9-914d0a6aeabd.png)

Reply:
I get this if I type "g" in the searcher

Reply:
Hmm okay I'll add this as a note for myself to look at